storagenode/monitor/monitor_test.go

```diff
@@ -42,13 +37,11 @@ func TestMonitor(t *testing.T) {
 		for _, storageNode := range planet.StorageNodes {
 			storageNode.Storage2.Monitor.Loop.TriggerWait()
 
-			info, err := satellite.Contact.Service.FetchInfo(ctx, storageNode.Local().Node)
-			require.NoError(t, err)
-
+			bw := storageNode.Local().Capacity.FreeBandwidth
 			stats, err := storageNode.Storage2.Inspector.Stats(ctx, &pb.StatsRequest{})
 			require.NoError(t, err)
 			if stats.UsedSpace > 0 {
-				assert.Equal(t, freeBandwidth-stats.UsedBandwidth, info.Capacity.FreeBandwidth)
+				assert.Equal(t, freeBandwidth-stats.UsedBandwidth, bw)
```

Comment:
nit - instead of `freeBandwidth` and `bw`, maybe `freeBandwidthOld` and `freeBandwidthNew` would be easier to understand names